common/containers/QrSignerModal/index.tsx

```diff
@@ -94,7 +94,7 @@ class QrSignerModal extends React.Component<Props, State> {
       return;
     }
 
-    this.props.finalizeSignature(signature);
+    this.props.finalizeSignature('0x' + signature);
```

Comment:
TODO (for myself): circle back and provide utility function for this.

Reply:
`addHexPrefix` does this from `ethereumjs-util`

Reply:
@maciejhirsz You cool with making this small tweak? No reason not to merge after imo.

Reply:
Sure!